src/platform/device.gni

```diff
@@ -119,6 +116,13 @@ declare_args() {
   }
 }
 
+declare_args() {
+  # Include wifi-paf to commission the device or not
+  # This is a feature of Wi-Fi spec that it can be enabled if wifi is enabled
+  # and the supplicant can support.
+  chip_device_config_enable_wifipaf = chip_enable_wifi && current_os == "linux"
+}
+
```

Comment:
```suggestion
```

Reply:
Please remove this per the TT discussion today, I'll help with a separate workflow to test this.